### PR TITLE
Specify: Bower version 1.6.5

### PR DESCRIPTION
**Pull request by andygout**
cc @keirog

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,6 +9,7 @@ clean:
 	git clean -fxd
 
 install:
+	npm install bower@1.6.5
```
Comment (matthew-andrews) [inline Makefile]: does this need to be in both makefile and package.json? why not package.json only?

**Comment by keirog** [comment]
Not sure this is necessary--see the [`test-upgrade` branch in circle](https://circleci.com/gh/Financial-Times/next-opt-in-out) (bumping the node version)

**Comment by andygnewman** [comment]
Agree with @keirog - Origami upped the bower version in OBT to 1.6.5 yesterday which resolved issues that we were having.

**Comment by andygout** [comment]
Agreed that upgrading node to v4.2.2 is the best solution ( @keirog 's branch below ) so am closing this branch:

https://github.com/Financial-Times/next-opt-in-out/tree/test-upgrade

**Pull request closed without merging**